tests/units/test_log_empty_message.rb

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env ruby
+require File.dirname(__FILE__) + '/../test_helper'
+
+class TestLogEmptyMessage < Test::Unit::TestCase
+  def setup
+    set_file_paths
+  end
+  
+  def test_repo_with_empty_commit
```

Comment:
I think this test name should be changed to `test_repo_with_empty_commit_message` since empty commit is something different.

For this functionality, I would think to do two tests:
1. 
a test that does not have a commit message and does not use the `:allow_empty_message` option
2. a test that does not have a commit message and DOES use the `:allow_empty_messages` option

Since the `:allow_empty` and `:allow_empty_message` options are so similar, I found the use of the `:allow_empty` option in this particular test confusing. I'll leave it up to you if you want to do anything about this.

Reply:
>I think this test name should be changed to `test_repo_with_empty_commit_message` since empty commit is something different.

oh, year, I did misspell here

>Since the :allow_empty and :allow_empty_message options are so similar, I found the use of the :allow_empty option in this particular test confusing. I'll leave it up to you if you want to do anything about this.

`:allow_empty` just allow commit without changes, so it was added to both commits

Reply:
>a test that does not have a commit message and does not use the `:allow_empty_message` option

we can't commit without this option, that result in error in `git`, and test fail with error

Reply:
So, as test without `:allow_empty_message` can't work because of `git` internal error, can we resolve this?